SECURITY.md

```diff
@@ -0,0 +1,29 @@
+# Security
+
+GitHub takes the security of our software products and services seriously, including all of the open source code repositories managed through our GitHub organizations, such as [GitHub](https://github.com/GitHub).
+
+Even though [open source repositories are outside of the scope of our bug bounty program](https://bounty.github.com/index.html#scope) and therefore not eligible for bounty rewards, we will ensure that your finding gets passed along to the appropriate maintainers for remediation.
```

Comment:
we can just mention that the finding will be passed appropriately, don't need to mention bug bounty

Reply:
Done e3c6164e73d90cbdd6e41d3732e0ed8720b4c164